battleship-client/src/main/java/com/academy/solid/nie/client/ui/BoardFX.java

```diff
@@ -67,11 +67,12 @@ private void createCellInRow(final EventHandler<? super MouseEvent> handler,
      * Indicates the cell as occupied by the ship.
      * The cell color changes to white with a green border.
      *
-     * @param ship - represents single instance of Ship
+     * @param ship  - represents single instance of Ship
      * @param point - represents single Point of the board
      */
     void markFieldAsOccupiedByShip(Ship ship, Point2D point) {
```

Comment:
Good idea to hide this in a `markField` method.

Reply:
Out of curiosity, I'd like knowing what will use it. Board has BoardFx field. Anything else, that warrants this being package private method?

Reply:
Only Board use this method

Reply:
Thank you.